### PR TITLE
Example like tags

### PR DESCRIPTION
**Pull request by trshemanske**
For this collection of tags, I used the same tag description for each since they are very closely related and it might help a user to understand that which of question or problem to use is largely a personal choice (at least IMHO :) )

### CODE REVIEW

File: packages/docs/pages/reference/elements/example.mdx [hunk 1]
```diff
@@ -8,12 +8,68 @@ import {
 
 # `<example>`
 
-Short description of what the `<example>` tag does.
+The `<example>`, `problem`, and `question` tags are closely related and have
+the same syntax.  The `title` element is optional but encouraged, and the
+structure can contain `hint`s, `answer`s, and `solution`s (in that order).
```
Comment (siefkenj) [inline packages/docs/pages/reference/elements/example.mdx:13]: Let's make sure tags are referred to using
```
[`<tag-name>`](tag-name)
```
syntax.

I think this is worth breaking into two paragraphs: the first explains what an example is for. The second explains that other elements are part of the `example-like` class of elements and share the structure of examples, but whose labels will be different.

File: packages/docs/pages/reference/elements/problem.mdx [hunk 2]
```diff
@@ -8,12 +8,50 @@ import {
...
\ No newline at end of file
+ ```ptx-example
+<FRAGMENT parents="p">
+<problem>
+  <title>Row and column spaces</title>
+  <p>
+  We know that the rank of a matrix <m>A</m> and of its row-reduced echelon
+  form <m>R</m> is the same.  Determine if the row space and column space of
+  <m>A</m> and <m>R</m> are always the same.
+  </p>
```
Comment (siefkenj) [inline packages/docs/pages/reference/elements/problem.mdx:29]: Interesting. I think this is not good form. You either use only `<p>`s or you use `<statement>`, `<hint>`, etc.

File: packages/docs/pages/reference/elements/question.mdx [hunk 3]
```diff
@@ -8,12 +8,47 @@ import {
...
 
 <SyntaxDisplay name="question" />
 
 ## Examples
 
-    
\ No newline at end of file
+ ```ptx-example
+<FRAGMENT parents="p">
+question>
```
Comment (siefkenj) [inline packages/docs/pages/reference/elements/question.mdx:23]: Malformed. Also example-like cannot be children of `<p>`

**Comment by trshemanske** [comment]
Hopefully files did not get tangled.  I had a issue with git, changing branches and stashing/popping files which led to some malevolent merge sequences :)

**Pull request merged**